packages/nextjs/app/builders/0xE00E720798803B8B83379720c42f7A9bE1cCd281/page.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import Image from "next/image";
+import { ExternalLink, Github, Twitter } from "lucide-react";
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const name = "Joe Leo";
+const nickname = "TheCoderJoe";
+const description =
+  "Web3 builder, Solidity developer, and AI explorer. I love creating decentralized solutions, collaborating with innovative minds, and pushing the boundaries of blockchain technology. Let's connect and build the future together!";
+const walletAddress = "0xE00E720798803B8B83379720c42f7A9bE1cCd281";
+const avatarUrl =
+  "https://res.cloudinary.com/dig0e9lmr/image/upload/v1747066063/anime-style-character-space_nm3mi9.jpg";
+
+const socialLinks = [
+  {
+    platform: "github",
+    url: "https://github.com/web-ghost-dotcom",
+    icon: <Github size={20} />,
+    color: "hover:text-purple-500",
+  },
+  {
+    platform: "BuidlGuidl",
+    url: "https://app.buidlguidl.com/builders/0xE00E720798803B8B83379720c42f7A9bE1cCd281",
+    icon: <Twitter size={20} />,
+    color: "hover:text-blue-400",
+  },
+];
+
+const JoeLeoWeb3Profile: NextPage = () => {
+  return (
+    <div className="w-full max-w-2xl mx-auto bg-gradient-to-br from-gray-900 to-gray-800 text-gray-100 rounded-2xl shadow-xl overflow-hidden transition-colors duration-200 border border-gray-700">
```

Comment:
Your page looks good in dark mode, but you might not have noticed the theme switcher at the bottom right corner. It would be great if your content adapted to the light theme as well. You might want to rebase your branch onto `origin/main` because we’ve fixed the `dark` variant, so it’ll be much easier to make it adaptive to the theme.